### PR TITLE
Catalog price rule - make active

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) addresses MAGEDOC-3990 and clarifies the process for creating a catalog price rule and using the change schedule to make it active.

Updated and added screen captures. 

## Affected documentation pages

- https://docs.magento.com/m2/ee/user_guide/marketing/price-rules-catalog-create.html
- https://docs.magento.com/m2/ee/user_guide/marketing/price-rules-catalog-scheduled-changes.html

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

### CODE REVIEW

File: src/marketing/price-rule-catalog-scheduled-changes.md [hunk 1]
```diff
@@ -3,12 +3,87 @@ conditions: Default.EE-B2B
...
+
+1. In the **Scheduled Changes** box at the top of the page, click <span class="btn">Schedule New Update</span>.
+
+1. With the **Save as a New Update** option selected, do the following:
+
+    - In the **Update Name** field, enter a name for the update to the rule.
+
+    - Enter a brief **Description** of the update, including how or why it is applied.
+
+    - Use the Calendar (![]({% link images/images/btn-calendar.png %}){: .Inline}) to choose the **Start Date** and **End Date** for the scheduled change to be in effect. To create an open-ended change, leave the End Date blank.
```
Comment (keharper) [inline src/marketing/price-rule-catalog-scheduled-changes.md]: Either bold the last instance of End Date or make it lowercase.

File: src/marketing/price-rule-catalog-scheduled-changes.md [hunk 2]
```diff
@@ -3,12 +3,87 @@ conditions: Default.EE-B2B
...
+
+    {:.bs-callout .bs-callout-info}
+    You cannot add an end date cannot later to a price rule that was originally created without one. In such a case, it is necessary to create a duplicate rule that includes the end date that is needed.
```
Comment (keharper) [inline src/marketing/price-rule-catalog-scheduled-changes.md]:
```suggestion
    You cannot add an end date to a price rule that was originally created without one. In such a case, it is necessary to create a duplicate rule that includes the end date that is needed.
```

File: src/marketing/price-rule-catalog-scheduled-changes.md [hunk 3]
```diff
@@ -3,12 +3,87 @@ conditions: Default.EE-B2B
...
+
+    ![Catalog price rules - new scheduled changes]({% link images/images-ee/price-rule-catalog-schedule-update.png %}){: .zoom}
+    _Add Scheduled Changes_
+
+1. Scroll down to the **Rule Information** section and make the changes for the rule.
+
+   You can schedule a change for any of the rule parameters, including the websites (scope)/customer groups for the rule, conditions of the rule, and actions applied by the rule. Fore more information, see [Creating a Catalog Price Rule]({% link marketing/price-rules-catalog-create.md %}).
```
Comment (keharper) [inline src/marketing/price-rule-catalog-scheduled-changes.md]:
```suggestion
   You can schedule changes for any rule parameter, including the websites (scope)/customer groups for the rule, conditions of the rule, and actions applied by the rule. For more information, see [Creating a Catalog Price Rule]({% link marketing/price-rules-catalog-create.md %}).
```

File: src/marketing/price-rule-catalog-scheduled-changes.md [hunk 4]
```diff
@@ -3,12 +3,87 @@ conditions: Default.EE-B2B
...
+
+1. With the **Save as a New Update** option selected, do the following:
+
+    - In the **Update Name** field, enter a name for the update to the rule.
+
+    - Enter a brief **Description** of the update, including how or why it is applied.
+
+    - Use the Calendar (![]({% link images/images/btn-calendar.png %}){: .Inline}) to choose the **Start Date** and **End Date** for the scheduled change to be in effect. To create an open-ended change, leave the End Date blank.
+
+    {:.bs-callout .bs-callout-info}
```
Comment (keharper) [inline src/marketing/price-rule-catalog-scheduled-changes.md]: Does merchdocs still use this construction for note blocks? In devdocs, it's just `{:.bs-callout-info}`

File: src/marketing/price-rule-catalog-scheduled-changes.md [hunk 5]
```diff
@@ -3,12 +3,87 @@ conditions: Default.EE-B2B
...
+
+    ![Catalog price rules - new scheduled changes]({% link images/images-ee/price-rule-catalog-schedule-update.png %}){: .zoom}
+    _Add Scheduled Changes_
+
+1. Scroll down to the **Rule Information** section and make the changes for the rule.
+
+   You can schedule a change for any of the rule parameters, including the websites (scope)/customer groups for the rule, conditions of the rule, and actions applied by the rule. Fore more information, see [Creating a Catalog Price Rule]({% link marketing/price-rules-catalog-create.md %}).
+
+    {:.bs-callout .bs-callout-info}
+    If you make a change to any of the rule information parameters, make sure that the **Status** is set as `Active` if you want the change to result in an actively applied rule.
```
Comment (keharper) [inline src/marketing/price-rule-catalog-scheduled-changes.md]: It's possible that you want to disable an existing rule during a campaign. So I would  rewrite the last clause to say make sure the status is correct.

File: src/marketing/price-rule-catalog-scheduled-changes.md [hunk 6]
```diff
@@ -3,12 +3,87 @@ conditions: Default.EE-B2B
...
+
+1. When complete, click <span class="btn">Save</span>.
+
+    The scheduled change appears at the top of the product page, with the start and end dates of the campaign.
```
Comment (keharper) [inline src/marketing/price-rule-catalog-scheduled-changes.md]: The product page? You're defining a price rule here.

File: src/marketing/price-rules-catalog-create.md [hunk 7]
```diff
@@ -84,7 +84,7 @@ Most of the available conditions are based on existing attribute values. To appl
 
       The selected condition appears in the statement, followed by two more bold links. The options differ depending on the condition attribute you select. The statement now says:
 
-         If **ALL** of these conditions are **TRUE**: </ br>Attribute Set **is** …
+        If **ALL** of these conditions are **TRUE**: <br/>Attribute Set **is** …
 
    - Click **is** and choose the comparison operator that describes the condition to be met. These options may include an option for different comparisons, such as matching values, not including or including at least one of a value, and great than, equal to, and less than a numerical amount. In this example, the options are `is` and `is not`.
```
Comment (keharper) [inline src/marketing/price-rules-catalog-create.md]: "greater than"

(When I use the suggestion tool, GH changes the spacing at the beginning of the paragraph.)

File: src/marketing/price-rules-catalog-create.md [hunk 8]
```diff
@@ -179,17 +181,25 @@ To apply the rule, do one of the following:
...
+    The Rule Information page displays an updated timeline in the Scheduled Changes for the rule.
+
+    ![Catalog price rules - scheduled changes]({% link images/images-ee/price-rule-scheduled-changes-updated.png %}){: .zoom}
+    _Scheduled Changes_
 
-2. Test the rule to make sure that it works correctly.
+1. Test the rule to make sure that it works correctly.
 
-  Price rules are automatically processed with other system rules each night. When you create a new price rule, allow enough time for it to get into the system. Then, test the rule to make sure that it works correctly. As new rules are added, Magento recalculates the prices and the priorities accordingly.
+    Price rules are automatically processed with other system rules each night. When you create a new price rule, allow enough time for it to get into the system. Then, test the rule to make sure that it works correctly. As new rules are added, Magento recalculates the prices and the priorities accordingly.
```
Comment (keharper) [inline src/marketing/price-rules-catalog-create.md]: I'm not so sure about this statement. The rules on my devbox take effect immediately, though I often reindex frequently when working on staging. I don't know who to ask to verify this.

Comment (leslietilling) [reply src/marketing/price-rules-catalog-create.md]: I am not sure either. This information was already here, but might need to be updated.

**Comment by leslietilling** [comment]
running tests

**Pull request merged**